tests/service/jobs/test_config.py

```diff
@@ -67,5 +66,6 @@ def test_delay_config_set_failure(svc_client_cache, it_remote_repo_url, view_use
 
     from renku.ui.service.jobs.delayed_ctrl import delayed_ctrl_job
 
-    with pytest.raises(MigrationRequired):
-        delayed_ctrl_job(context, view_user_data, job.job_id, renku_module, renku_ctrl)
+    updated_job = delayed_ctrl_job(context, view_user_data, job.job_id, renku_module, renku_ctrl)
```

Comment:
I believe this test will fail once we increase metadata version, right? This isn't very nice but we can fix it then.

Reply:
yes, now there's two repos, one with old metadata and one with new one, and we need to update the new one when we update the metadata. But that's much better than what it was, where a lot of these tests just checked that "MigrationRequired" is raised and not that setting the config or creating the dataset etc. works

Reply:
I think it makes sense to have integration tests that work on an actual repo and having to keep that up to date isn't great but also not a big deal